ds3-autogen-c/src/main/java/com/spectralogic/ds3autogen/c/models/Type.java

```diff
@@ -51,4 +51,26 @@ public String getUnqualifiedName() {
     public String getNameUnderscores() {
         return helper.camelToUnderscore(getUnqualifiedName());
     }
+
+    public String generateMatcher() {
+        String output = new String();
+        final int numConstants = this.enumConstants.size();
+        for (int currentIndex = 0; currentIndex < numConstants; currentIndex++) {
+            output += "    ";
```

Comment:
You could use the CHelper.indent(1) here (if you make it public) to lessen the likelihood of mismatched indentations within the generated code.